### PR TITLE
fix(curriculum): use python syntax for >= & <=

### PR DESCRIPTION
**Pull request by jip2587**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This pull request changes the symbols for "greater than or equal to" and "less than or equal to" from normal mathematical symbols into their python syntax. The python syntax is used for "equal to" and "not equal to" earlier in the table, and I think it makes sense to be consistent here. Note this is my first pull-request ever, so I'd appreciate coaching if I've made any errors with it. Thanks!

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/655a2a7210094920069b117c.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ Python has the following comparison operators:
       <td>Less than</td>
     </tr>
     <tr>
-      <td>&ge;</td>
+      <td>>=</td>
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/655a2a7210094920069b117c.md:40]:
```suggestion
      <td>&gt;=</td>
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/655a2a7210094920069b117c.md [hunk 2]
```diff
@@ -37,11 +37,11 @@ Python has the following comparison operators:
...
       <td>Greater than or equal to</td>
     </tr>
     <tr>
-      <td>&le;</td>
+      <td><=</td>
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/655a2a7210094920069b117c.md:44]:
```suggestion
      <td>&lt;=</td>
```

**Review by majestic-owl448 — COMMENTED** [review]
The html entities are needed to avoid parsing issues of the angular brackets, if the "less/greater than or equal" symbols are to be like in the code, they still need the angular brackets to be html entities

**Comment by jip2587** [comment]
Ah, right, good call. I've committed the suggestions. Thanks!

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**